.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets":[
+        "@babel/preset-env"
+    ],
+    "plugins": [
+        ["@babel/transform-runtime"],
+        ["istanbul"]
+    ]  
+}
```

Comment:
Why is this touched for a task that is adding docker?

Reply:
I did not touch this